{"name": "Ошибка авторизации несуществующего курьера", "status": "passed", "description": "Проверка ошибки 404 при несуществующем курьере", "steps": [{"name": "Отправка запроса авторизации несуществующего курьера", "status": "passed", "start": 1752420316846, "stop": 1752420317084}, {"name": "Проверка, что возвращается ошибка 404", "status": "passed", "start": 1752420317084, "stop": 1752420317084}], "start": 1752420316846, "stop": 1752420317084, "uuid": "67638c30-3efa-4135-bcb5-d72a5aa21973", "historyId": "567cdf5b9ac7920ad3a7b27f2decce19", "testCaseId": "567cdf5b9ac7920ad3a7b27f2decce19", "fullName": "Tests.test_login_courier.TestLoginCourier#test_login_with_nonexistent_courier", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-T2S04D8"}, {"name": "thread", "value": "8088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login_courier"}]}